multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/flowable/ResumeProcessActionTest.java

```diff
@@ -0,0 +1,38 @@
+package org.cloudfoundry.multiapps.controller.process.flowable;
+
+import static org.mockito.Mockito.times;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.multiapps.controller.process.variables.Variables;
+import org.flowable.engine.runtime.Execution;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class ResumeProcessActionTest extends ProcessActionTest {
+
+    @Test
+    void testResumeExecutionWithExecutionsAtReceiveTask() {
+        processAction.execute("fake-user", PROCESS_GUID);
+        Mockito.verify(flowableFacade, times(2))
```

Comment:
Why is the resume action called twice here?

Reply:
For the root process and one of the parent processes